protos/mocap/mocap.proto

```diff
@@ -0,0 +1,153 @@
+syntax = "proto3";
+
+package mavsdk.rpc.mocap;
+
+option java_package = "io.mavsdk.mocap";
+option java_outer_classname = "MocapProto";
+
+/**
+ * Motion Capture allow vehicles to navigate when a global
+ * position source is unavailable or unreliable
+ * (e.g. indoors, or when flying under a bridge. etc.).
+ */
+service MocapService {
+    // Send Global position/attitude estimate from a vision source.
+    rpc SetVisionPositionEstimate(SetVisionPositionEstimateRequest) returns(SetVisionPositionEstimateResponse) {}
+    // Send motion capture attitude and position.
+    rpc SetAttitudePositionMocap(SetAttitudePositionMocapRequest) returns(SetAttitudePositionMocapResponse) {}
+    // Send odometry information with an external interface.
+    rpc SetOdometry(SetOdometryRequest) returns(SetOdometryResponse) {}
+}
+
+message SetVisionPositionEstimateRequest {
+    VisionPositionEstimate vision_position_estimate = 1;
+}
+message SetVisionPositionEstimateResponse {
+    MocapResult mocap_result = 1;
+}
+
+message SetAttitudePositionMocapRequest {
+    AttitudePositionMocap attitude_position_mocap = 1;
+}
+message SetAttitudePositionMocapResponse {
+    MocapResult mocap_result = 1;
+}
+
+message SetOdometryRequest {
+    Odometry odometry = 1;
+}
+message SetOdometryResponse {
+    MocapResult mocap_result = 1;
+}
+
+// Global position/attitude estimate from a vision source.
+message VisionPositionEstimate {
+    uint64 time_usec = 1; // Position frame timestamp UNIX Epoch time (0 to use Backend timestamp)
+    Position position = 2; // Global position (m)
+    float roll = 3; // Roll angle (rad)
```

Comment:
Please add the unit.

Reply:
(e.g. `roll_rad` similar to [here](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L196))